src/add-patient-link.tsx

```diff
@@ -2,12 +2,11 @@ import React from 'react';
 import { ConfigurableLink } from '@openmrs/esm-framework';
 import Add20 from '@carbon/icons-react/es/add/20';
 import { HeaderGlobalAction } from 'carbon-components-react/es/components/UIShell';
-import './add-patient-link.css';
 export default function Root() {
   return (
     <HeaderGlobalAction aria-label="Add" aria-labelledby="Add Patient" name="AddPatientIcon">
-      <ConfigurableLink to="${openmrsSpaBase}/patient-registration">
-        <Add20 className="add-patient-color" />
+      <ConfigurableLink style={{ color: '#FFFFFF' }} to="${openmrsSpaBase}/patient-registration">
```

Comment:
looks good,did we have a ticket Id for this fix?

Reply:
Hello, @HerbertYiga,
This is related to the [MF-464](https://issues.openmrs.org/browse/MF-464), to add the Add-Patient button on the primary navbar.
I did that but you would have noticed that the color for that was blue.
So this is to make the color white for that.
Thank you.